src/modules/lighthouse/auditUrl.ts

```diff
@@ -0,0 +1,38 @@
+import ora from 'ora'
+import chalk from 'chalk'
+
+import log from '../../logger'
+import { lighthouse, workspaces } from '../../clients'
+import { getAccount, getWorkspace } from '../../conf'
+import { TableGenerator } from './TableGenerator'
+
+const [account, currentWorkspace] = [getAccount(), getWorkspace()]
+
+async function isProdutionWorkspace(): Promise<boolean> {
+  const meta = await workspaces.get(account, currentWorkspace)
+  return meta.production
+}
+
+export default async (url: string, option: any) => {
+  if (await isProdutionWorkspace()) {
+    log.error(`You can not run lighthoust audits on production workspaces.`)
+    return
+  }
+
+  const spinner = ora(`Running Lighthouse on url: ${chalk.blue(url)}`).start()
+  try {
+    const report = await lighthouse.runAudit(url)
+    spinner.stop()
+
+    if (option.j || option.json) {
```

Comment:
No need for `option.j || `options.json` anymore, now it can be only `option.json`

Reply:
Great!